src/messages.js

```diff
@@ -5,36 +5,68 @@ module.exports = Tag => ({
+        'Tu léxico es tan elevado que ni un bot te puede entender :jenkins_triggered:',
     ],
     "lookup_challengers onbehalf_missing": [
         'Creo que preguntas por alguien más, pero no entiendo quién.',
         'Parece que quieres preguntar en lugar de otra persona, pero no entiendo bien a quién.',
         '¿Estás preguntando por alguien más?, no estoy seguro de entender. :thinking_face:',
         'Que te valga verga morro.',
+        'Si vas a tirar una indirecta al menos dí a quién.',
     ],
     "lookup_challengers myself no_coins": [
         'Parece que ya no te quedan monedas, así que no puedes retar nadie. :disappointed:',
         'Veo que no te quedan monedas... ya no puedes retar nadie. :disappointed:',
         'Sin monedas no puedes retar a nadie, ni modo así son las reglas. :disappointed:',
         'No traes nada... de monedas :mepanchea:',
+        'No te quedan monedas, mejor ponte a trabajar',
+        'Tienes tantas monedas como Pancho dinero en su cuenta de banco... 0 :mepanchea:',
+        'Ya jugaste demasiado esta semana, no tienes llenadera :disapproval:',
+        'Ya no tienes monedas, no te cansas de perder verdad?',
+        'Hasta pareciera que te pagan por jugar, ya no puedes retar esta semana.',
+        'A Nadie',
+        'Tienes tantas monedas como nivel en Smash... 0 :squidbaging: ',
+        'No tienes monedas, pero siempre se pueden armar las amistosas :eyes:',
+        'Ya te quedaste sin monedas, no estes de insolente :face_with_rolling_eyes:',
+        'Que triste que tengas tantas ganas de jugar pero ya no puedas',
     ],
     "lookup_challengers onbehalf no_coins": [
         Tag `Parece que a ${'user'} no le quedan monedas, así que no puede retar nadie. :disappointed:`,
         Tag `Veo que a ${'user'} no le quedan monedas... ya no puede retar nadie. :disappointed:Tag `,
         Tag `Sin monedas ${'user'} no puede retar a nadie, ni modo así son las reglas. :disappointed:`,
-        Tag `*Que te valga verga morrro, tu no traes nada*, ${'user'} no tiene monedas... :mepanchea:`
+        Tag `*Que te valga verga morrro, tu no traes nada*, ${'user'} no tiene monedas... :mepanchea:`,
+        Tag `Se acabaron los puntos gratis que daba ${'user'}, mejor suerte la proxima semana`,
+        Tag `A ${'user'} ya le dieron su arrastrada semanal, vuelva prontos`,
+        Tag `${'user'} ya no tiene monedas, por lo tanto no puede jugar`,
     ],
     "lookup_challengers myself_all": [
         Tag `Estos son los jugadores que puedes retar:\n\n${'listOfValidPlayers'}.`,
         Tag `Revisando la tabla, veo que estos son los que puedes retar:\n\n${'listOfValidPlayers'}.`,
         Tag `${'listOfValidPlayers'}\n\nLa lista de arriba muestra a quiénes puedes retar.`,
         Tag `Ni les vas a ganar, pero a los jugadores que puedes retar son:\n\n${'listOfValidPlayers'}.`,
+        Tag `You expected the challenge list? too bad!\n https://tenor.com/FlMC.gif`,
```

Comment:
La trolleada del día. No se que opine @Xotl

Reply:
Me agrada la trolleada... aunque creo que no se va a renderear el Gif porque internamente se usan bloques. -> https://api.slack.com/tools/block-kit-builder?blocks=[{"type"%3A"section"%2C"text"%3A{"type"%3A"mrkdwn"%2C"text"%3A"You expected the challenge list%3F too bad!\n https%3A%2F%2Ftenor.com%2FFlMC.gif"}}]

Reply:
Podemos dejarlo para una issue diferente y que acepte este tipo de mensajes.

Reply:
¿entonces lo dejamos así y luego lo corregimos?

Reply:
Ya creamos la issue. @FranciscoOnt, ¿podrías remover el GIF? Lo vamos a trabajar en otra issue.

Reply:
@FranciscoOnt, nomás quita ese mensaje ya para mergear el PR.

Reply:
listo jefe